app/src/main/kotlin/de/janhopp/luebeckmensawidget/api/MensaApi.kt

```diff
@@ -16,9 +18,10 @@ class MensaApi(
         get() = currentTime.mensaApiFormat
 
     suspend fun getMealsToday(): List<MensaDay> = runCatching {
-        client.get("https://speiseplan.mcloud.digital/meals?date=$date")
+        client.get("https://speiseplan.mcloud.digital/v2/meals?location=ME&date=$date")
```

Comment:
It probably should be "location=ME,CA", because the Mensa and Cafeteria are treated as different locations.

Reply:
The widget filtered this before as well and only ever showed Mensa meals

Reply:
Ohhh